css-fonts/styles.css

```diff
@@ -0,0 +1,7 @@
+h1,h2 {
+font-family: 'Baloo Thambi 2', cursive;
```

Comment:
In CSS, the selector is not indented in any way, but each property is indented by one tab